packages/nc-gui/components/smartsheet/Grid.vue

```diff
@@ -325,14 +362,21 @@ useEventListener(document, 'keyup', async (e: KeyboardEvent) => {
 
 /** On clicking outside of table reset active cell  */
 const smartTable = ref(null)
-onClickOutside(smartTable, () => {
+onClickOutside(smartTable, (e: PointerEvent) => {
```

Comment:
is it necessary to add `e: PointerEvent`? Should be typed without having to add the event type yourself.